gremlin-dotnet/glv/GraphTraversal.template

```diff
@@ -68,7 +68,7 @@ namespace Gremlin.Net.Process.Traversal
         public GraphTraversal< <%= method.t1 %> , <%= method.t2 %> > <%= toCSharpMethodName.call(method.methodName) %><%= method.tParam %> (<%= method.parameters %>)
         {
         <%  if (method.parameters.contains("params ")) {
-      %>    var args = new List<object>(<%= method.paramNames.init().size() %> + <%= method.paramNames.last() %>.Length) {<%= method.paramNames.init().join(", ") %>};
+      %>    var args = new List< <%= method.argsListType %> >(<%= method.paramNames.init().size() %> + <%= method.paramNames.last() %>.Length) {<%= method.paramNames.init().join(", ") %>};
```

Comment:
Can we do something about the extra spacing?

Reply:
+1

Reply:
Agree, those spaces shouldn't be there and I just found a way to remove them: Variables can be used in a Groovy template in the form of `$variable` which avoids the problem with the double `<` as `<<%= variable %>>` doesn't work.
I pushed a commit that should remove all those unnecessary spaces from the generated classes in Gremlin.Net (at least I hope that I got all).

Reply:
Wow, nice!
I've reviewed the end result and looks good.